tests/conftest.py

```diff
@@ -59,6 +61,14 @@ def _get_latest_commit():
     return commit
 
 
+@pytest.fixture(scope="module")
+def setup_api_server():
+    yield
+    for path in (os.environ["DSGRID_QUERY_OUTPUT_DIR"], os.environ["DSGRID_API_SERVER_STORE_DIR"]):
+        if os.path.exists(path):
+            shutil.rmtree(path)
```

Comment:
Should there be an option to leave these files for debugging purposes?

Reply:
I typically set breakpoints that get triggered before this occurs. I'm sure that what you're asking can be done with pytest, but I would have to investigate.